src/chrome/content/rules/3DCenter.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="3DCenter.org (cacert)" platform="cacert">
+	<target host="www.3dcenter.org" />
+	<target host="m.3dcenter.org" />
```

Comment:
add
    <target host="3dcenter.org" />
    <rule from="^http://3dcenter\.org/" to="https://www.3dcenter.org/"/>